hyp3_gamma/insar/getDemFileGamma.py

```diff
@@ -10,4 +10,4 @@ def get_dem_file_gamma(dem_image: str, dem_par: str, safe_dir: str, pixel_size:
     with NamedTemporaryFile() as dem_tif:
         prepare_dem_geotiff(dem_tif.name, geometry, pixel_size)
         execute(f'dem_import {dem_tif.name} {dem_image} {dem_par} - - $DIFF_HOME/scripts/egm2008-5.dem '
-                f'$DIFF_HOME/scripts/egm2008-5.dem_par - - - 1', uselogging=True)
+                f'$DIFF_HOME/scripts/egm2008-5.dem_par - - - - -', uselogging=True)
```

Comment:
`-` just means "use the default value", there's no need to provide trailing `-` parameters if we're not actively setting any later parameters
```suggestion
                f'$DIFF_HOME/scripts/egm2008-5.dem_par', uselogging=True)
```

Reply:
The call of the dem_import is:
dem_import <input_DEM> <DEM> <DEM_par> [input_type] [priority] [geoid] [geoid_par] [geoid_type] [latN_shift] [lonE_shift] [zflg] [no_data]
I didn't check if we do not provide [no_data] option, what the result will be. I suggest keep the [no_data] option, so that we get clear understanding which option is which when we compare with the call format of the command dem_import.